Doc/library/enum.rst

```diff
@@ -55,7 +55,7 @@ helper, :class:`auto`.
 
 .. class:: auto
 
-    Instances are replaced with an appropriate value for Enum members.
+    Instances are replaced with an appropriate value for Enum members. Initial value starts at 1.
```

Comment:
You need two spaces after the period. I'm now thinking that this should say "By default, the initial value starts at 1", since it's possible to override it.

Reply:
@ericvsmith Someone needs to go through the `Enum` docs and rewrite them.  If no one else does I will later this year.  For inspiration:

  https://www.divio.com/blog/documentation/

Reply:
@ericvsmith: Sorry I jumped the gun on that one.  If @YoSTEALTH  make that last change feel free to commit it.

Reply:
I am unable to edit this file, its already merged? do i have to recreate the pull request?

Reply:
@YoSTEALTH : It's been merged. I don't think it's super-important to fix, but if you want to create a new PR I'll merge it.

Reply:
@ericvsmith i recreated it https://github.com/python/cpython/pull/17878